### ISSUE TITLE
direnv instead of dir-locals?

### ISSUE THREAD

**Issue by benkolera** [benkolera/spacemacs-hie-nix#9]
dir-locals keep having weird bugs for me anyway.

See https://github.com/haskell/haskell-ide-engine/issues/117#issuecomment-430034902

**Comment by arjunv27** [comment]
Worth checking out: 
https://github.com/target/lorri

**Comment by benkolera** [comment]
Yeah, I already have and it works really well. Sadly hie is still a bit too
slow on obelisk projects so I haven't been able to get much serious use out
of it. :(

I forgot to put a note on this repo to deprecate this idea and do a PR to
the hie docs. Thanks for the poke. :)
